test/clean.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+docker images -q 'librarytest/*' | xargs docker rmi -f
```

Comment:
Can we do a little changing here, to get rid of tags instead of images?  Images will go away, but only if nothing else is using it.
```bash
docker images 'librarytest/*' | awk 'NR>1 { print $1":"$2 }' | xargs -r docker rmi
```